instrumentation/opentelemetry-instrumentation-pymemcache/tests/test_pymemcache.py

```diff
@@ -59,7 +59,7 @@ class PymemcacheClientTestCase(
 
     def setUp(self):
         super().setUp()
-        PymemcacheInstrumentor().instrument()
+        PymemcacheInstrumentor().instrument(sanitize_query=False)
```

Comment:
The default behavior shouldn't be False? to keep the same behavior.
this could cause different results for different versions
And according to the specs the Requirement Level for db.statement is "if not explicitly disabled via instrumentation configuration." I think the sanitizing should be the same

Reply:
@shalevr can you link the spec where it says this

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#call-level-attributes

The third note referred to the db.statement, but it is not explicitly written what to do in case of sanitization.
In general, I think that it is better to sanitize the value by default but I'm not sure if we can change the default behavior like this